packages/chaire-lib-backend/src/config/server.config.ts

```diff
@@ -50,6 +50,25 @@ export type TrRoutingConfig = {
     // support it in both batch and single calculation
 };
 
+export type OsrmRoutingConfig = {
+    /**
+     * Set to null when using remote osrm server
+     */
+    port: number | null;
+    /**
+     * Set to null when using remote osrm server
+     */
+    osrmPath: string | null;
```

Comment:
Can we change this to hostname ?

Or have both an OSRMPath and hostname field ? (Or name it OsrmPathOrHostname)

Reply:
I renamed it to osrmPathOrHostname